docs/tutorial/profiling-by-example.rst

```diff
@@ -1,14 +1,14 @@
 .. meta::
-   :description: Omniperf: Profiling by example
-   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD
+   :description: ROCm Compute Profiler: Profiling by example
+   :keywords: ROCm Compute Profiler, ROCm, profiler, tool, Instinct, accelerator, AMD
 
 ********************
 Profiling by example
 ********************
 
 The following examples refer to sample :doc:`HIP <hip:index>` code located in
-:fab:`github` :dev-sample:`ROCm/omniperf/blob/amd-mainline/sample <>` and distributed
-as part of Omniperf.
+:fab:`github` :dev-sample:`ROCm/rocprofiler-compute/blob/amd-mainline/sample <>`
+and distributed as part of ROCm Compute Profiler.
 
 .. include:: ./includes/valu-arithmetic-instruction-mix.rst
```

Comment:
Seem to be several references to omniperf commands in the include files here. I couldn't quite track down where they came from (i..e omniperf profile -n instmix --no-roof -- ./instmix)

Reply:
Updated in a7161d6